rfcs/041-item-statuses/README.md

```diff
@@ -0,0 +1,311 @@
+    <td>In cataloguing</td>
+    <td>47</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td><code>!</code></td>
+    <td>On holdshelf</td>
+    <td>33</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td><code>e</code></td>
+    <td>On exhibition</td>
+    <td>24</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td><code>s</code></td>
+    <td>On search</td>
+    <td>16</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td><code>z</code></td>
+    <td>Claims returned</td>
+    <td>10</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td><code>x</code></td>
+    <td>Withdrawn</td>
+    <td>10</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td><code>a</code></td>
+    <td>a</td>
+    <td>3</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td><code>d</code></td>
+    <td>On display</td>
+    <td>3</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td><code>r</code></td>
+    <td>Restricted</td>
+    <td>3</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td><code>t</code></td>
+    <td>In quarantine</td>
+    <td>2</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td><code>f</code></td>
+    <td>Returned to vendor</td>
+    <td>1</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td><code>0</code></td>
+    <td>0</td>
+    <td>1</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td><code>q</code></td>
+    <td>Test record</td>
+    <td>1</td>
+    <td></td>
+  </tr>
+</table>
+
+
+
+## Proposal
+
+We will not expose the Sierra item statuses.
+Instead, we will create a new set of statuses, which will be mapped as follows:
+
+<table>
+  <tr>
+    <th>id</th>
+    <th>label</th>
+    <th>Sierra statuses</th>
+    <th>comment</th>
+  </tr>
+  <tr>
+    <td>available</td>
+    <td>Available</td>
+    <td><code>-</code> / Available</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td>closed</td>
+    <td>Closed</td>
+    <td>
+      <code>h</code> / Closed <br/>
+    </td>
+    <td>
+      We distinguish between closed/restricted/unavailable in the access statuses.
+      I'm assuming it's a useful distinction here, but we could collapse this into "Unavailable" if not.
+    </td>
+  </tr>
+  <tr>
+    <td>restricted</td>
+    <td>Restricted</td>
+    <td>
+      <code>6</code> / Restricted <br/>
+    </td>
+    <td>
+      Ditto above.
+    </td>
+  </tr>
+  <tr>
+    <td>unavailable</td>
+    <td>Unavailable</td>
+    <td>
+      <code>r</code> / Unavailable <br/>
+      <code>m</code> / Missing <br/>
+      <code>x</code> / Withdrawn
+    </td>
+    <td>
+      I assume we wouldn't want to advertise if an item was missing.
+    </td>
+  </tr>
+  <tr>
+    <td>on-exhibition</td>
+    <td>On exhibition</td>
+    <td>
+      <code>e</code> / On exhibition
+    </td>
+    <td>
+    </td>
+  </tr>
+  <tr>
+    <td>on-display</td>
+    <td>On display</td>
+    <td>
+      <code>d</code> / On display
+    </td>
+    <td>
+    </td>
+  </tr>
+  <tr>
+    <td>in-cataloguing</td>
+    <td>In cataloguing</td>
+    <td>
+      <code>p</code> / In cataloguing
+    </td>
+    <td>
+      This feels analogous to the "on order" or "on exhibition" statuses.
```

Comment:
Indeed and also all of them have an alarmingly large overlap with their equivalent locationTypes. Another example of something we should try to rationalise, rather than display the same info twice.